src/component/calculator.css

```diff
@@ -9,7 +9,6 @@
   border: 1px solid black;
   display: flex;
   flex-direction: column;
-  width: 45rem;
 }
 
```

Comment:
![calculator](https://user-images.githubusercontent.com/93039359/199879379-e1cce0ad-9e62-4c2e-89b7-15a864a5355d.PNG)

**[Optional]**

- [ ] You have made your readme descriptive 😊 it would be nice if you add a license file in your project because you have a link to your license file in your readme file as you can see in the image ⬆️You can add a license file in your project using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 👍